homeassistant/components/wmspro/scene.py

```diff
@@ -0,0 +1,58 @@
+"""Support for scenes provided by WMS WebControl pro."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from wmspro.scene import Scene as WMS_Scene
```

Comment:
```suggestion
from wmspro.scene import Scene as WebControlScene
```
Rather call it something else than Pascal_Case